tests/zeroconfig/windows/testdata/expected_resource_metrics.yaml

```diff
@@ -0,0 +1,6 @@
+resource_metrics:
+  - attributes:
+    scope_metrics:
+      - metrics:
+          - name: otelcol_exporter_sent_spans
```

Comment:
Do you mind adding all the expected metrics including labels? `<ANY>` can be used as a value for dynamic ones where we mainly care about the attr presence/key.

Reply:
> Do you mind adding all the expected metrics including attributes*?

Most of the metrics are independent of the .NET instrumentation, or do you mean only the ones expected after the request to the instrumented application is done?

Reply:
Yes, this is an internal collector metric. The SFx.NET metrics are disabled by default so I didn't want to rely on them.

Reply:
I have a story to add the resource spans to testutils, the plan is to switch this test to actually check for the spans.

Reply:
Adding resource spans checks sounds great though it took me a bit to understand the evaluated flow here. You're installing two collectors, one for zero config w/ ingest url configured to be the signalfx receiver of the second collector, which is configured to report to the test's otlp sink?

Reply:
yes, I tried to document that on the test code, may require some re-wording to make that clear... see https://github.com/signalfx/splunk-otel-collector/pull/2901/files#diff-164dad5cd795a3f716eea69cefc15b2062e4d9d766cfa9ee103ae943716a40f1R41